docs/api-reference/react/deckgl.md

```diff
@@ -62,6 +62,12 @@ Default: If the `views` property is not supplied, deck.gl will create a full scr
 
 > deck.gl will render all the views in the provided order. This may not matter as depth testing is enabled by default by deck.gl (unless transparent layers are being rendered, in which case rendering order starts to matter again), but is useful when rendering 2D layers and disabling depth testing.
 
+##### `controller` (Function | Boolean | Object, optional)
```

Comment:
Should we still support this global `controller` prop? I am thinking we could delete it, and instead specify that the `Deck` component creates a default `MapView({controller: true})` if the `views` prop is not provided.

That way, if one doesn't want a MapController, one has to pass in a `views=[new MapView()]`


If we keep it should it be supported also by `Deck`? If so should we move it to the `Deck` dock, which is "inherited" by `DeckGL`.
?

Reply:
Will we stop supporting the old use case (DeckGL as child of MapGL)? I personally like the flat props (longitude, latitude, controller etc.) as they do not require importing another symbol (MapView).

Reply:
Aren't those independent issues? 

`DeckGL` can still build a `viewState` from longitude etc regardless of whether we support a `controller` prop?